gemrb/core/Map.h

```diff
@@ -583,8 +588,8 @@ class GEM_EXPORT Map : public Scriptable {
 	void SortQueues();
 	//Actor* GetRoot(int priority, int &index);
 	void DeleteActor(int i);
-	void Leveldown(unsigned int px, unsigned int py, unsigned int& level,
-		Point &p, unsigned int& diff);
+	void Leveldown(unsigned int x, unsigned int y, unsigned int &cost,
+				   Point &newPoint, unsigned int &costDiff);
```

Comment:
no need here either as it doesn't even reach 100, but I know you mostly just renamed the vars in this case.

Reply:
However, the definition of the function wasn't committed or updated for these names.